Makefile

```diff
@@ -275,7 +275,6 @@ gh-pages:
 		-e '/default\.js/d' \
 		-e '/utils\/ios-overlap-fix\.js/d' \
 		-e "s/var cacheVersion = '0';/var cacheVersion = '`git rev-parse HEAD`';/g" \
-		-e 's|utils/watchwatchers\.js|lib/watchwatchers.js|' \
```

Comment:
This conflicts with my changes (already removed there). Can we wait for #3820 to be merged?